CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [unreleased]
+
+### Fixed
+
+- Android: Entering Picture-in-Picture automatically from all screens when navigating the app to the background (after navigating to Basic Picture-in-Picture screen once)
```

Comment:
This entry is very specific for our samples, is this something that only happens in our samples? If so we should make this clear. Otherwise, can we make it a bit more generic?

Reply:
It's kind of both, I would suppose that it should happen to other single activity apps. I'll write it in a more generic way.

Reply:
https://github.com/bitmovin/bitmovin-player-react-native/pull/424/commits/132dbd68bb0f1cb2e3b5105ecdf45e5362088f59